exporter/splunkhecexporter/metricdata_to_splunk_test.go

```diff
@@ -30,8 +30,6 @@ import (
 )
 
 func Test_metricDataToSplunk(t *testing.T) {
```

Comment:
Can you please add tests for the new functionality?

Reply:
done